packages/salesforcedx-utils-vscode/test/unit/helpers/utils.test.ts

```diff
@@ -7,20 +6,99 @@
  */
 
 import { expect } from 'chai';
-import { extractJsonObject } from '../../../src/helpers';
+import * as fs from 'fs';
+import * as sinon from 'sinon';
+import {
+  extractJsonObject,
+  flushFilePath,
+  flushFilePaths
+} from '../../../src/helpers';
 
 describe('getConfigSource', () => {
-
   it('should extract a JSON string from larger string and then return as an object', async () => {
-
-    const exampleJsonString = JSON.stringify({ name: 'exampleName', error: 'exampleError' });
+    const exampleJsonString = JSON.stringify({
+      name: 'exampleName',
+      error: 'exampleError'
+    });
     const exampleString = `junk text <junk +text junk text ${exampleJsonString} junk text junk text junk text`;
 
     const testParse = extractJsonObject(exampleString);
 
     expect(testParse.name).to.equal('exampleName');
     expect(testParse.error).to.equal('exampleError');
+  });
+});
+
+describe('flushFilePath', () => {
+  it('should call fs.realpathSync.native() to resolve a path', async () => {
+    const filePath = 'c:\\Users\\temp\\exampleFile.js';
+    const realpathSyncNativeStub = sinon
+      .stub(fs.realpathSync, 'native')
+      .returns(filePath);
+
+    const result = flushFilePath(filePath);
+
+    expect(realpathSyncNativeStub.called).to.equal(true);
```

Comment:
Since calledOnce is used below, is the expect for "called" needed here?

Reply:
+1 nice catch